src/colvarproxy_io.cpp

```diff
@@ -0,0 +1,291 @@
+// -*- c++ -*-
+
+// This file is part of the Collective Variables module (Colvars).
+// The original version of Colvars and its updates are located at:
+// https://github.com/Colvars/colvars
+// Please update all Colvars source files before making any changes.
+// If you wish to distribute your changes, please submit them to the
+// Colvars repository at GitHub.
+
+// Using access() to check if a file exists (until we can assume C++14/17)
+#if !defined(WIN32) || defined(__CYGWIN__)
+#include <unistd.h>
+#endif
+#if defined(WIN32)
+#include <io.h>
+#endif
+
+#include <cerrno>
+#include <cstdio>
+
+#include <list>
+#include <map>
+#include <sstream>
+#include <fstream>
+
+#include "colvarmodule.h"
+#include "colvarproxy_io.h"
+
+
+colvarproxy_io::colvarproxy_io()
+{
+  input_buffer_ = NULL;
+  restart_frequency_engine = 0;
+  input_stream_error_ = new std::istringstream();
+  input_stream_error_->setstate(std::ios::badbit);
+}
+
+
+colvarproxy_io::~colvarproxy_io()
+{
+  delete input_stream_error_;
```

Comment:
Good point, memory leaks for from this class would be rare, but still they should be deleted behind the scenes since they are traditional pointer without garbage collection (no point in commenting further about smart pointers).